wls-ergebnismeldung-service/src/test/java/de/muenchen/oss/wahllokalsystem/ergebnismeldungservice/rest/begruendung/BegruendungControllerTest.java

```diff
@@ -0,0 +1,88 @@
+package de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.rest.begruendung;
+
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.domain.BezirkUndWahlIDStapelart;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.domain.Stapelart;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.service.begruendung.BegruendungModel;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.service.begruendung.BegruendungReference;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.service.begruendung.BegruendungService;
+import lombok.val;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+
+@ExtendWith(MockitoExtension.class)
+class BegruendungControllerTest {
+
+    @Mock
+    BegruendungService begruendungService;
+
+    @Mock
+    BegruendungDTOMapper begruendungDTOMapper;
+
+    @InjectMocks
+    BegruendungController unitUnderTest;
+
+    @Nested
+    class GetBegruendung {
+
+        @Test
+        void should_returnDTOWithHttpStatusOk_when_serviceReturnedData() {
+            val wahlID = "wahlID";
+            val wahlbezirkID = "wahlbezirkID";
+
+            val begruendungReference = new BegruendungReference(wahlbezirkID, wahlID, Stapelart.LTW_BZW_A);
+            val bezirkUndWahlIDStapelart = new BezirkUndWahlIDStapelart("wahlbezirkID", "wahlID", Stapelart.LTW_BZW_A);
+            val mockedServiceResponse = new BegruendungModel("wahlbezirkID", "wahlID", Stapelart.LTW_BZW_A, null, null, true, true);
```

Comment:
```suggestion
            val bezirkUndWahlIDStapelart = new BezirkUndWahlIDStapelart(wahlbezirkID, wahlID, Stapelart.LTW_BZW_A);
            val mockedServiceResponse = new BegruendungModel(wahlbezirkID, wahlID, Stapelart.LTW_BZW_A, null, null, true, true);
```
wenn du die variablen schon erstellt hast solltest du sie hier denke ich auch übergeben und den wert nicht hart rein schreiben